src/test/java/com/powsybl/openloadflow/OpenLoadFlowParametersTest.java

```diff
@@ -164,7 +164,7 @@ void testMaxIterationReached() {
         parameters.setWriteSlackBus(true);
         Network network = EurostagTutorialExample1Factory.create();
         LoadFlow.Runner loadFlowRunner = new LoadFlow.Runner(new OpenLoadFlowProvider(new SparseMatrixFactory()));
-        network.getGenerator("GEN").setTargetV(5);
+        network.getVoltageLevelStream().forEach(vl -> vl.setNominalV(21));
```

Comment:
Why ?

Reply:
That was just a way to get max iteration reached, as setting a target value too far from nominal voltage is not possible anymore. I improved it with a clearer and cleaner change, and added a comment.